draft-ietf-httpbis-http2bis.xml

```diff
@@ -538,7 +538,7 @@ HTTP Frame {
         </t>
         <aside>
           <t>
-            Note: Previous versions of this specification used the term "header block" in place of
+            Note: <xref target="RFC7540">RFC7540</xref> used the term "header block" in place of
```

Comment:
```suggestion
            Note: <xref target="RFC7540">RFC 7540</xref> used the term "header block" in place of
```
In prose, the name has a space (it should be non-breaking, but I don't tolerate `&#xa0;` very well.